core/platform/models.py

```diff
@@ -87,7 +88,7 @@ def import_models(cls, model_names):
                 from core.storage.activity import gae_models as activity_models
                 returned_models.append(activity_models)
             elif name == NAMES.app_feedback_report:
-                from core.storage.app_feedback_report import gae_models as app_feedback_report_models # pylint: disable=line-too-long
+                from core.storage.app_feedback_report import gae_models as app_feedback_report_models  # isort:skip pylint: disable=line-too-long
```

Comment:
Mistake, removed.

Reply:
I think it's still there, in the other places.

Reply:
Oh, sorry, forgot to delete the comment.

Should be fixed now. I actually decided to refactor this file instead, because I also wanted to get rid of the pylint disables. In this place I removed the `as …` part, from the functional perspecitive it doesn't make any difference. From the perspective of readability I also think this change is fine, because we always use the import on the next line.

I also renamed core/platform/cloud_translate to core/platform/translate, so that it is more in line with naming of the other platform folders, the 'cloud' defines more specific implementation and in the translate folder there are both the cloud and dev implementation.

Reply:
Sgtm. Thanks!